let promise =new Promise((resolve,reject)=>{
    resolve("success")

    reject("an error have occured")
})

console.log(promise)